src/components/table/table-header/table-header.component.js

```diff
@@ -155,15 +155,22 @@ class TableHeader extends React.Component {
     } else {
       contents = this.props.children;
     }
+
+    const headerProps = {
+      align: this.props.align,
+      sortable: this.props.sortable
+    };
+
+    if (!Number.isNaN(Number(this.props.width))) headerProps.width = this.props.width;
```

Comment:
This should take the string just fine, it'll convert it

Reply:
I've had to revert back to the regex implementation as I need it to prevent any hex coercions as well